xicam/Acquire/devices/fastccd.py

```diff
@@ -10,7 +10,6 @@
 from ophyd import Component as Cpt
 from ophyd.device import FormattedComponent as FCpt
 from ophyd import AreaDetector
-from ophyd.utils import set_and_wait
```

Comment:
`set_and_wait` is still needed. Why was it removed?

Reply:
good question, by accident, can't explain when that happened